xsnippet/api/resources/syntaxes.py

```diff
@@ -0,0 +1,23 @@
+"""
+    xsnippet.api.resources.syntaxes
+    -------------------------------
+
+    The module implements the resource, that allows to retrieve
+    the list of supported snippet syntaxes.
+
+    :copyright: (c) 2017 The XSnippet Team, see AUTHORS for details
+    :license: MIT, see LICENSE for details
+"""
+
+from .. import resource
+from ..application import endpoint
+
+
+@endpoint('/syntaxes', '1.0')
+class Syntax(resource.Resource):
```

Comment:
Maybe call it `Syntaxes`? the same way as the endpoint?

Reply:
Will do!